examples/paymentbot.py

```diff
@@ -31,7 +33,7 @@ def start_with_shipping_callback(bot, update):
     description = "Payment Example using python-telegram-bot"
     # select a payload just for you to recognize its the donation from your bot
     payload = "Custom-Payload"
-    # get your provider_token at @botfather, see https://core.telegram.org/bots/payments#getting-a-token
+    # to get your provider_token see https://core.telegram.org/bots/payments#getting-a-token
```

Comment:
If we're into grammar, "In order to get a provider_token ..." would be more appropriate.